pyorientAssignment.py

```diff
@@ -36,14 +36,41 @@
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
 
+price = []
 for record in records:
-	print record.price
+   price.append(record.price)
+   
+totalPrice = 0
+
+for i in price:
+    totalPrice += i
+
+averagePrice = totalPrice / len(records)
+print 'average price: ' + str(averagePrice)
+
+highPrice = 500000000000000000
```

Comment:
you should set this to a low number initially, so that when you reach a higher price in the records it is replaced with the higher value.